### PR TITLE
Add project proposal

### PR DESCRIPTION
**Pull request by manav2401**
Added project proposal for ePBS implementation with inclusion list. 

This is a common proposal among me, [Anshal](https://github.com/anshalshukla), [Chirag](https://github.com/Chirag018), and [Ella](https://github.com/0xfmoi) as of now. 

Also, I merged few things from Ella's proposal but I am in favour of this new file name so for now have removed her file.

### CODE REVIEW

File: projects/epbs-with-inclusion-list.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+This requires changes in both the consensus as well as the execution layer. Potuz and Terrence have been working on the consensus specs and the plan is to fully understand them, review them and implement them in the Prysm consensus client. For execution layer, the specs (and an EIP) will be written at a later stage (and I hopefully plan to contribute to writing that as well) and the chosen client is Geth. The plan is to break down the whole implementation into different components and iteratively build out the required feature. Moreover, with ePBS, an inclusion list design proposed [here](https://ethresear.ch/t/no-free-lunch-a-new-inclusion-list-design/16389) will also be implemented for preventing censorship as both of them are complementary to each other.
```
Comment (taxmeifyoucan) [inline projects/epbs-with-inclusion-list.md:25]: I see you elaborate on chosen clients here, you can still mention it above so it's clear. 

One thing to add here is maybe being bit more specific on what will be your part. It's a big project so I wonder whether you have some specific focus or you can just mention you will do general contributions at first.

File: projects/epbs-with-inclusion-list.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+### Fellows 
+
+*Are there any fellows working with you on this project?*
+
+As it's a huge project, yes there are other fellows also working on this with me. 
+- [Anshal Shukla](https://github.com/anshalshukla)
+- [Chirag Mehta](https://github.com/Chirag018)
+- [NC](https://github.com/naviechan) (An external participant)
+- [Ella](https://github.com/0xfmoi) (Haven't discussed with her yet, but saw her proposal on the same topic)
```
Comment (taxmeifyoucan) [inline projects/epbs-with-inclusion-list.md:68]: I see her proposal overlaps with yours and if you are going to collaborate, I would prefer merging them in favor of yours.

**Review by taxmeifyoucan — COMMENTED** [review]
Thanks for adding the proposal! It's great, I just nitpicked some small things. The main part would be to figure out overlap with @0xfmoi because we would end up with 2 very similar proposals.

**Comment by manav2401** [comment]
@taxmeifyoucan thanks for the suggestions. I got to know about Ella's proposal when I was done writing mine so just went ahead and created a PR. I'll definitely be in touch and try to modify the existing proposal in that case. Thank you!

**Comment by manav2401** [comment]
@taxmeifyoucan I've addressed the comments and also merged the proposals. Looking for reviews. Thanks!

**Pull request merged**